jac-uk/apply
# Fix the broken preview URLs and workflow

## What's included?
This PR fixes the previews workflow so preview urls are created & updated

## Who should test?
✅ Developers

## How to test?
Commit a change to this branch. Check that the preview workflow runs successfully and the preview url is updated (see that the comment timestamp and commit hash are up to date)

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this effects package-lock
